test/e2e/fishapp/certs/spire_setup.yaml

```diff
@@ -0,0 +1,307 @@
+apiVersion: v1
```

Comment:
Once we introduce a helm chart for SPIRE, it'd be good to integrate that into the tests instead